package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "swr-openapi",
-  "version": "4.1.2",
+  "type": "module",
+  "version": "5.0.0-rc.14",
```

Comment:
Are the versions managed manually or by automation. If manually you probably want to set this to 5.0.0 (i.e. drop the rc)

Reply:
I'll ensure this is updated once the PR merges to main.